app/adapters/application.js

```diff
@@ -1,94 +1,48 @@
-import DS from 'ember-data';
+import JSONAPIAdapter from '@ember-data/adapter/json-api';
 import { inject as service } from '@ember/service';
+import { later } from '@ember/runloop';
 
-export default DS.JSONAPIAdapter.extend({
-  defaultSerializer: 'JSONAPISerializer',
-  intl: service(),
-  toaster: service(),
+export default class ApplicationAdapter extends JSONAPIAdapter {
+  @service intl;
+  @service toaster;
 
-  translateAndParseSuccesType(type) {
-    const singular = type.slice(0, -1);
-    return this.intl.t(singular).toLowerCase();
-  },
+  // eslint-disable-next-line no-unused-vars
+  async ajax(url, method) {
+    try {
+      if (['POST', 'DELETE'].includes(method)) { // methods that cause unwanted effects when executing a request multiple times
```

Comment:
PUT ook ? die van agendaitem documents

Reply:
Dat is er inderdaad ook eentje die je geen twee keer zou willen uitvoeren, maar die loopt niet via ember-data, dus hoort niet thuis in dit rijtje.